src/Evolution/DiscontinuousGalerkin/Limiters/Weno.hpp

```diff
@@ -339,6 +340,21 @@ bool Weno<VolumeDim, tmpl::list<Tags...>>::operator()(
       modified_neighbor_solutions.insert(
           std::make_pair(neighbor, interpolant.interpolate(data.volume_data)));
     }
+
+    // Then the correction is added one tensor component at a time:
+    const auto wrap_mean_correction = [&mesh, &modified_neighbor_solutions ](
+        auto tag, const auto& tensor) noexcept {
```

Comment:
Since `tensor` is a pointer, I would have found `const auto` easier to read and less redundant since we don't need pointers to pointers here :)